src/three-render-objects.js

```diff
@@ -185,11 +190,10 @@ export default Kapsule({
 
   stateInit: () => ({
     scene: new three.Scene(),
-    camera: new three.PerspectiveCamera(),
```

Comment:
The camera declaration should remain in `stateInit`. Otherwise you may run into situations where you try to use it before it's declared, for example by invoking `cameraPosition` before initialization.

If you need one of the config options you can always get it as input to `stateInit`:
```js
stateInit: ({ cameraType }) => ({
  camera: /* cameraType logic */
  ...
}) 
```

Reply:
done